cache/hhvm-scalable-cache.h

```diff
@@ -969,6 +972,14 @@ FastHashMonitor(){
     // TODO @ Ziyue: how to become 'optimal' instead of only better than baseline
     auto delta = (baseline_performance_with_threshold - performance)/baseline_performance_with_threshold * thput_step / baseline_step;
     performance_depletion += delta; // INTEGRATION to help decide whether to stop FH
+
+    printf("thput_step: %lu, ", thput_step);
+    printf("thput_step: %lu, ", thput_step);
+    printf("baseline_step: %lu, ", baseline_step);
+    printf("threshold: %.3lf, ", baseline_performance_with_threshold);
+    printf("performance: %.3lf, ", performance);
+    printf("delta: %.3lf, ", delta);
+    printf("depleted: %.3lf\n", performance_depletion);
```

Comment:
Hopefully, my debug statements didn't/don't affect latency measurements, because these will all print every 0.1s. I think they're really useful for parsing through the output and making sure it works as expected but might be a factor when doing official benchmarking to keep in mind (either comment out or keep for both in old vs new).
Maybe we can make the printf's toggleable via #define, idk, just a heads up